lib/api/objectPut.js

```diff
@@ -99,19 +99,24 @@ function objectPut(authInfo, request, streamingV4Params, log, callback) {
             // have overwritten if there was an existing version or object
             const oldByteLength = objMD && objMD['content-length']
```

Comment:
Can you add a TODO note in there mentioning the follow up. We can easily lose track of this otherwise.